src/euphorie/client/utils.py

```diff
@@ -219,6 +220,20 @@ def logoMode(self):
         return 'alien' if 'alien' in self.extra_css else 'native'
 
     @reify
+    def styles_override(self):
+        css = ""
+        if IItaly.providedBy(self.request):
+            css = """
+#steps .topics .legend li.answered.risk::before {
+    background: purple;
+}
+#steps .topics .questions li.answered.risk a::before {
+    background: purple;
+}
```

Comment:
Why not add the country to the body as a class (like `body class="country-it"`) and use that in the standard CSS?

Reply:
Yes, would have been an option too